client/main/dataset.js

```diff
@@ -455,10 +456,20 @@ const DataSetModel = Backbone.Model.extend({
             levels = new Array(columnPB.levels.length);
             for (let i = 0; i < levels.length; i++) {
                 let levelPB = columnPB.levels[i];
-                levels[i] = {
-                    label: levelPB.label,
-                    value: levelPB.value,
-                };
+                if (column.measureType === 'nominaltext') {
+                    levels[i] = {
+                        label: levelPB.label,
+                        value: i,
+                        importValue: levelPB.importValue
+                    };
+                }
+                else {
+                    levels[i] = {
+                        label: levelPB.label,
+                        value: levelPB.value,
+                        importValue: levelPB.value.toString()
```

Comment:
oh righto. i thought it was going over the wire, but this is entirely living on the client side.